### DISCUSSION TITLE
When we upgrade trino version we got an error version showed from monitor ui

### DISCUSSION THREAD

**Discussion by ImTangYun** [trinodb/trino#23252] [Q&A]
Recently, we upgrade our trino version from 363 to 433, we found that the version showed from monitor ui is still 363:
![image](https://github.com/trinodb/trino/assets/5945486/4eec0350-acf8-4674-9cd1-f37f55f6ee06)
which may be expected as:
![image](https://github.com/trinodb/trino/assets/5945486/6f67c8a8-9375-4264-952e-635ec362707d)

How we merge our code with trino version 433?
1.git checkout 368
2.git checkout -b our_368
3.add some commit to meet our needs
4.git checkout 433
5.git checkout -b our_433
6.git log our_368 and found the oldest commit id of our commit(for example：1053684f388ea68e8e9fc618579867201a8e032b)
7.git rebase --onto our_433 1053684f388ea68e8e9fc618579867201a8e032b
8.fix some conflicts and git rebase --continue
9.Then our_433 is what we compile and deploy

**Comment by ImTangYun** [comment]
We fixed This problem in pr:https://github.com/trinodb/trino/pull/20050

**Comment by electrum** [comment]
The reported Trino version is the same version shown when running `git describe`. It is based on Git tags, so you need to fetch tags to have the version shown correctly: `git fetch upstream --tags`

(replace `upstream` with the `git remote` name for the upstream Trino repository)

**Comment by electrum** [comment]
You can run `git tag` and verify that `433` is in the list of tags. Then run `git describe` and make sure that it shows the version that you expect.

**Comment by ImTangYun** [comment]
> You can run `git tag` and verify that `433` is in the list of tags. Then run `git describe` and make sure that it shows the version that you expect.

Ok Thanks!
